site/src/pages/DeploySettingsPage/Option.tsx

```diff
@@ -91,11 +91,11 @@ export const OptionValue: FC<OptionValueProps> = (props) => {
                 }}
               >
                 {isEnabled && (
-                  <CheckCircleOutlined
+                  <LabelImportant
```

Comment:
UI design feedback: I get what the icon change is going for, but I think it's mixing "UI metaphors" a little bit

With the label icon, I almost always see it applied for actual labels, like in Gmail:
![Screenshot 2024-04-16 at 12 32 40 PM](https://github.com/coder/coder/assets/28937484/e885f9b8-fd55-434d-833e-819797514f35)

My worry is that if a user sees a "label-ish" element, they're going to think that it can be interacted with like a label. I also think most of our "stylized bullet points" tend to be circles, too

If we're trying to go for something that is less positive than a green checkmark, but still feels experimental, and is a circle, then maybe the build circle icon would be a good bet?

![Screenshot 2024-04-16 at 12 38 07 PM](https://github.com/coder/coder/assets/28937484/3bac513e-ab86-49d7-b431-7b14310576e0)

Also, just in case, if you didn't know, MUI [has an icon preview](https://mui.com/material-ui/material-icons) you can use to help find the perfect icon

Reply:
Michael brings up a good point. Just adding that I really like your idea to get rid of `CheckCircleOutlined` - that is a good call!

Reply:
This is a great call-out, thanks @Parkreiner! Definitely agree with the above, and that icon indeed looks better.